src/Pos/Delegation/Listeners.hs

```diff
@@ -82,7 +87,7 @@ pskLightRelay = Data $ DataParams $ \pSk -> do
 pskHeavyRelay
     :: WorkMode ssc ctx m
     => Relay m
-pskHeavyRelay = Data $ DataParams $ handlePsk
+pskHeavyRelay = Data $ DataParams MsgTransaction $ \_ _ -> handlePsk
```

Comment:
@georgeee take a look please, are you sure it should be `MsgTransaction` for heavyweight delegation?